bot/cogs/tags.py

```diff
@@ -9,15 +9,22 @@
 )
 
 from bot.constants import (
-    ADMIN_ROLE, MODERATOR_ROLE, OWNER_ROLE,
-    SITE_API_KEY, SITE_API_TAGS_URL, TAG_COOLDOWN,
+    ADMIN_ROLE, BOT_COMMANDS_CHANNEL, DEVTEST_CHANNEL,
+    HELPERS_CHANNEL, MODERATOR_ROLE, OWNER_ROLE,
+    SITE_API_KEY, SITE_API_TAGS_URL, TAG_COOLDOWN
 )
 from bot.decorators import with_role
 from bot.exceptions import CogBadArgument
 from bot.pagination import LinePaginator
 
 log = logging.getLogger(__name__)
 
+TEST_CHANNELS = [
+    DEVTEST_CHANNEL,
+    BOT_COMMANDS_CHANNEL,
+    HELPERS_CHANNEL
+]
+
```

Comment:
i know that people usually use lists for this anyways, but, since we're not modifying it anyways, why not make it a tuple?